src/quo/components/wallet/account_card/view.cljs

```diff
@@ -96,87 +95,86 @@
+          {:customization-color customization-color
+           :border-radius       16
+           :theme               theme
+           :pressed?            pressed?}])
+       [rn/view {:style style/profile-container}
+        [rn/view {:style {:padding-bottom 2 :margin-right 2}}
+         [text/text {:style style/emoji} emoji]]
+        [rn/view {:style style/watch-only-container}
+         [text/text
+          {:size            :paragraph-2
+           :weight          :medium
+           :number-of-lines 1
+           :max-width       110
+           :margin-right    4
+           :ellipis-mode    :tail
+           :style           (style/account-name type theme)}
+          name]
+         (when watch-only? [icon/icon :i/reveal {:color colors/neutral-50 :size 12}])
+         (when missing-keypair?
+           [icon/icon :i/alert {:color (properties/alert-icon-color theme) :size 12}])]]
+       [text/text
+        {:size   :heading-2
+         :weight :semi-bold
+         :style  (style/account-value type theme)}
+        balance]
+       (when metrics?
+         [rn/view {:style style/metrics-container}
+          [metrics-percentage type theme percentage-value]
+          (when (not= :empty type)
+            [metrics-info type theme amount])])
+       (when watch-only?
+         [gradient-overview theme customization-color])])))
 
 (defn- add-account-view
-  [_]
-  (let [pressed? (reagent/atom false)]
-    (fn [{:keys [on-press customization-color theme metrics?]}]
-      [rn/pressable
-       {:on-press     on-press
-        :on-press-in  #(reset! pressed? true)
-        :on-press-out #(reset! pressed? false)
-        :style        (style/add-account-container {:theme    theme
-                                                    :metrics? metrics?
-                                                    :pressed? @pressed?})}
-       [button/button
-        {:type                :primary
-         :size                24
-         :icon                true
-         :accessibility-label :add-account
-         :on-press            on-press
-         :pressed?            @pressed?
-         :on-press-in         #(reset! pressed? true)
-         :on-press-out        #(reset! pressed? false)
-         :customization-color customization-color
-         :icon-only?          true}
-        :i/add]])))
+  [{:keys [on-press customization-color metrics?]}]
+  (let [theme                  (quo.theme/use-theme-value)
+        [pressed? set-pressed] (rn/use-state false)
+        on-press-in            (rn/use-callback #(set-pressed true))
+        on-press-out           (rn/use-callback #(set-pressed false))]
+    [rn/pressable
+     {:on-press     on-press
+      :on-press-in  on-press-in
+      :on-press-out on-press-out
+      :style        (style/add-account-container {:theme    theme
+                                                  :metrics? metrics?
+                                                  :pressed? pressed?})}
```

Comment:
btw the reason this is a pressable wrapping a button is to expand the styling beyond the button, which is centered in the component. 
as far as I recall the thing is that the border of this component also has some pressed state styling to it so it needs to be aware of when it's pressed in etc as well.
Perhaps there was something more to it but pretty sure it was along that lines of why this was handled this way.

Reply:
i just removed on-press handlers from the button

Reply:
yeah makes sense, I wonder does that affect the styles of the button in pressed state then.